### DISCUSSION TITLE
I can't start mssql?

### DISCUSSION THREAD

**Discussion by herrwinfried** [containers/podman#16638] [Q&A]
Hi, I'm using OpenSUSE Tumbleweed. I want to use MSSQL but after installing it Status is "Exited".

_Firewalld is turned off in the system._

The commands I type in order
```
sudo zypper dup -y
sudo zypper in -y podman\* 
sudo systemctl enable --now podman.service podman.socket
systemctl --user enable --now podman.service podman.socket
```
```
podman pull mcr.microsoft.com/mssql/server:2019-latest
podman volume create mssql_data
podman run -d -e 'ACCEPT_EULA=Y' -e 'MSSQL_SA_PASSWORD=5sd4f6sf&%' --name MSSQL -p 1460:1433 -v mssql_data:/var/mssql/data mcr.microsoft.com/mssql/server:2019-latest
```
[MSSQL Log(podman)](https://github.com/containers/podman/files/10098215/podman.txt)

❯ podman --version
podman version 4.2.1

Kernel: 6.1.0-rc6-2.g62cc237-default

**Comment by vrothberg** [comment]
I see a number of permission denied errors which can have many sources.  Checkout @rhatdan's blog on how to diagnose these errors: https://www.redhat.com/sysadmin/container-permission-denied-errors

Ultimately, it doesn't look like a Podman issue.

**Reply by rhatdan** [reply]
You might need to add a :U option on the volume mount to make podman chown the directory to match the UID of the mariadb process within the container.

**Reply by herrwinfried** [reply]
hello, I did a reinstall with a clean install, this time I didn't use the 6.1-rc Kernel and it worked.  The problem is most likely due to the Kernel under development.

Thank you to everyone who took the time.
